src/components/common/TopNavbar.tsx

```diff
@@ -5,10 +5,96 @@ import { useCallback } from 'react'
 
 interface TopNavbarProps {
   isAdmin: boolean
+  isApplicant: boolean
   image: string
 }
 
-const TopNavbar: React.FC<TopNavbarProps> = ({ isAdmin, image }) => {
+// Admin Users: Users, Events, Calendar, Recruitment
+const admin = [
+  {
+    href: '/',
+    title: 'Home',
+  },
+  {
+    href: '/users',
+    title: 'Users',
+  },
+  {
+    href: '/events',
+    title: 'Events',
+  },
+  {
+    href: '/calendar',
+    title: 'Calendar',
+  },
+  {
+    href: '/recruitment',
+    title: 'Recruitment',
+  },
+]
+
+const applicants = [
+  {
+    href: '/application-status',
+    title: 'Status',
+  },
+  {
+    href: '/tasks',
+    title: 'Tasks',
+  },
+  {
+    href: '/recruitment/update-info',
```

Comment:
from the applicants perspective, i feel like we do not need to put the '/recruitment' infront as they won't need that prefix path since their routes in the site are rather limited already, can just be '/update-info', because i believe from the tab, there is no recruitment tab for the applicants also.